packages/frontend/amp/components/Ad.tsx

```diff
@@ -37,6 +37,28 @@ const adStyle = css`
     }
 `;
 
+const adClass = css`
+    display: none;
+`;
+
+const usAdRegionClass = css`
+    .amp-geo-group-us & {
```

Comment:
How do these classes get added/appear, and on which markup?

Reply:
They get added to the amp-ad element on server side. See line 134 of this file.